fe/fe-core/src/main/java/com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java

```diff
@@ -104,6 +107,14 @@
 
 public class MaterializedViewAnalyzer {
 
+    private static final Set<Table.TableType> SUPPORTED_TABLE_TYPE =
+            ImmutableSet.of(Table.TableType.OLAP,
+                    Table.TableType.HIVE,
+                    Table.TableType.HUDI,
+                    Table.TableType.ICEBERG,
+                    Table.TableType.JDBC,
+                    Table.TableType.MYSQL);
```

Comment:
how does SR perceive new data of JDBC/MYSQL table?

Reply:
we don't guarantee the freshness of jdbc table